File: (1)ServerCoreModel/GameServer/NetObject.cpp
#include "pch.h"
#include "NetObject.h"

NetObject::NetObject(UINT64 ownerId, Vector2 pos, float rot, UINT16 objectCode)
	: _ownerId(ownerId), _pos(pos), _rot(rot), _objectCode(objectCode)
{
}

File: Unity/Assets/Network/NetObject.cs
using System;
using System.Collections;
using System.Collections.Generic;
using CSharpServerCore;
using UnityEngine;

public class NetObject : MonoBehaviour
{
    protected  UInt16 netEnum;

    protected UInt64 ownerId;
    protected bool isMine;

    protected virtual void Awake()
    {

    }
    protected virtual void Start()
    {

    }


    public virtual void Initialize(UInt64 ownerId)
    {
        this.ownerId = ownerId;
        isMine = NetworkManager.instance.playerId == ownerId;
    }

    public bool IsMine
    {
        get
        {
            return isMine;
        }
    }
    public UInt64 OwnerId
    {
        get
        {
            return ownerId;
        }
    }
}